gallery/src/ThemeSettingsModal/ThemeSettingsModal.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useEffect, useState } from 'react';
+
+import {
+  NxButton,
+  NxModal,
+  NxFormGroup,
+  NxToggle,
+  NxFieldset,
+  NxRadio,
+  NxTooltip,
+  NxFontAwesomeIcon,
+  NxCode,
+  useToggle
+} from '@sonatype/react-shared-components';
+
+import { faInfoCircle } from '@fortawesome/free-solid-svg-icons';
+
+import { themingEnabled, themeOverride, setThemingEnabled, setThemeOverride } from '../themeClassManager';
+
+type Props = {
```

Comment:
Small thing but we could also move this into its own `types.ts` to follow our pattern

Reply:
Ops nvm these aren't entirely consistent in the gallery.